### ISSUE TITLE
paper questions

### ISSUE THREAD

**Issue by hezhu1996** [youngwanLEE/CenterMask#47]
@youngwanLEE Hi Youngwan, Jongyoul, thank for your greate work of CenterMask. However, I have a few questions which confused me a lot, Hope you can comment on it :)

Firstly,  from my understanding, you guys use the result of FCOS's bounding boxes as the input of ROIAlign, Are you using all k=100 predicted results or "**highest-scoring boxes**" to feed intothe SAG-mask branch for training mask branch?  like how do you define highest-scoring boxes, maybe a hyper-parameter to use top n results, or choose a confidence threshold that greater than m?

Secondly, In the inference stage, I noticed that adding mask head boosts AP from 37.8% to 38.3%, Do you use the original FCOS output to evaluate it or **re-predict** everything according to the mask-head?  I'm curious is it because the mask-head that refine the original FOCS bbox to make it better or because the mask head helps the LOSS to coverage better than before(if using FCOS to predict bbox).  
I'm only talking about AP_bbox not AP_mask, thank you very much!
